ompi/mca/osc/rdma/osc_rdma_dynamic.c

```diff
@@ -468,12 +468,9 @@ int ompi_osc_rdma_find_dynamic_region (ompi_osc_rdma_module_t *module, ompi_osc_
                      " (len %lu)", base, base + len, (unsigned long) len);
 
     OPAL_THREAD_LOCK(&module->lock);
-    // Make sure region isn't being touched.
-    ompi_osc_rdma_lock_acquire_exclusive (module, peer, offsetof (ompi_osc_rdma_state_t, regions_lock));
     if (!ompi_osc_rdma_peer_local_state (peer)) {
         ret = ompi_osc_rdma_refresh_dynamic_region (module, dy_peer);
         if (OMPI_SUCCESS != ret) {
-            ompi_osc_rdma_lock_release_exclusive (module, peer, offsetof (ompi_osc_rdma_state_t, regions_lock));
```

Comment:
There's an OPAL_THREAD_LOCK() acquired on line 470 above..should that get released here?